howso/utilities/installation_verification.py

```diff
@@ -645,13 +641,13 @@ def check_save(*, registry: InstallationCheckRegistry,
             source_df, _ = generate_dataframe(client=client)
         features = infer_feature_attributes(source_df)
         feature_names = list(features.keys())
-        trainee_obj = Trainee(
-            f"installation_verification check save ({get_nonce()})",
+        trainee = dict(
+            name=f"installation_verification check save ({get_nonce()})",
             features=features
         )
```

Comment:
Better to just pass these as direct parameters to create_trainee.

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60